lib/menu.rb

```diff
@@ -0,0 +1,47 @@
+# # As a customer
+# # So that I can check if I want to order something
+# # I would like to see a list of dishes with prices
+# # FOOD,MENU,ORDER?TEXT?
+# # As a customer
+# # So that I can order the meal I want
+# # I would like to be able to select some number of several available dishes
+# #
+# # As a customer
+# # So that I can verify that my order is correct
+# # I would like to check that the total I have been given matches the sum of the
```

Comment:
Metrics/LineLength: Line is too long. [81/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)